test/conv2d_bwd_data/main.cpp

```diff
@@ -0,0 +1,300 @@
+#include "config.hpp"
+#include "device.hpp"
+#include "host_tensor.hpp"
+#include "host_tensor_generator.hpp"
+#include "host_conv.hpp"
+#include "tensor_layout.hpp"
+#include "device_tensor.hpp"
+#include "device_conv_bwd_data.hpp"
+#include "element_wise_operation.hpp"
+#include "reference_conv_bwd_data.hpp"
+
+using F16  = ck::half_t;
+using F32  = float;
+using BF16 = ushort;
+using INT8 = int8_t;
+namespace ck {
+namespace tensor_operation {
+namespace device {
+namespace device_conv2d_bwd_data_instance {
+
+using DeviceConvBwdDataNoOpPtr =
+    DeviceConvBwdDataPtr<ck::tensor_operation::element_wise::PassThrough,
+                         ck::tensor_operation::element_wise::PassThrough,
+                         ck::tensor_operation::element_wise::PassThrough>;
+template <>
+void add_device_conv2d_bwd_data_xdl_nhwc_kyxc_nhwk_instances(std::vector<DeviceConvBwdDataNoOpPtr>&,
```

Comment:
same as above, just use simple function for "add_" instead of using template

https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/6d4450ef155c39af9ede2cd171be40ee06db9939/profiler/include/profile_conv_fwd_impl.hpp#L21

Reply:
done